living_documentation_regime/model/consolidated_issue.py

```diff
@@ -159,12 +158,8 @@ def generate_page_filename(self) -> str:
             md_filename_base = f"{self.number}_{self.title.lower()}.md"
             page_filename = sanitize_filename(md_filename_base)
         except AttributeError:
-            logger.error(
-                "Issue page filename generation failed for Issue %s/%s (%s). Issue does not have a title.",
-                self.organization_name,
-                self.repository_name,
-                self.number,
-                exc_info=True,
+            self.__errors.update(
+                {"AttributeError": "Issue page filename generation failed (issue does not have a title)."}
```

Comment:
Add some context info to be able to identify the issue as the source of problem.

Reply:
Please see the comment: https://github.com/AbsaOSS/living-doc-generator/pull/79#discussion_r1949080083.